README.md

```diff
@@ -30,10 +28,16 @@ These images are based off the [v2.1_beta5 release]((https://github.com/MTG/esse
 ### Git Nightly
 For a more up-to-date version of essentia we provide the `mtgupf/essentia-nightly` image. This is updated periodically
 based on the most recent commit on the `master` branch of essentia.
-We provide only one tag, based on Ubuntu 18.04 LTS:
+We provide the `latest` tag, based on Ubuntu 18.04 LTS:
 
   *  `latest` ([Dockerfile.ubuntu18.04-nightly](https://github.com/MTG/essentia-docker/blob/master/nightly/Dockerfile.ubuntu18.04-nightly))
 
+Following tags are available for using essentia with [tensorflow-support](https://mtg.github.io/essentia-labs//news/2020/01/16/tensorflow-models-released/).
```

Comment:
Add this as a separate header for Tensorflow support. Put the link to the essentia-labs page here.